src/main/scala/com/twitter/finagle/postgres/PostgresClient.scala

```diff
@@ -83,6 +83,7 @@ object PostgresClient {
     Types.TID -> TypeSpecifier("tidrecv", "tid"),
     Types.XID -> TypeSpecifier("xidrecv", "xid"),
     Types.CID -> TypeSpecifier("cidrecv", "cid"),
+    Types.JSON -> TypeSpecifier("json_recv", "json"),
```

Comment:
In general I've tried to only include types in the default map that are baked-in to postgres without installing an extension. I think that's now the case for `json` so this should be OK. Can you confirm?